docs/hopsworksai/user_management.md

```diff
@@ -0,0 +1,81 @@
+# User management
+In Hopsworks.ai users can be grouped into *organizations* to access the same resources.
+When a new user registers with Hopsworks.ai a new organization is created. This user later on can
+invite other registered users to their organization so they can share access to the same cluster.
+
+Configuration is also shared among users of the same organization. So if user Alice has configured
+her account with her credentials, Bob doesn't have to. Credits and cluster usage are also grouped to
+ease reporting.
+
+## Adding members to an organization
+Organization membership can be edited my clicking **Members** on the left of Hopsworks.ai Dashboard page.
+
+<p align="center">
+  <figure>
+    <a  href="../../../assets/images/hopsworksai/members_empty.png">
+      <img src="../../../assets/images/hopsworksai/members_empty.png" alt="Organization membership">
+    </a>
+    <figcaption>Organization membership</figcaption>
+  </figure>
+</p>
+
+To add a new member to your organization add the user's email and click **Add**
+
+!!! note
+     The invited user must have already been registered in Hopsworks.ai
```

Comment:
Are you sure of that? I thought it sent an email to the user if they were not already registered.

Reply:
I should rephrase it. Yes it will send the welcome email but still the user has to register to hopsworks.ai